Runtime error while running the 'daytona code' command and choosing WSname from the UI

**Describe the bug**
This bug was probably introduced in version v0.22.0. 
Running the `daytona code` command without workspace name argument leads to choosing the workspace from the UI. After choosing the workspace this error is thrown:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x269619a]

goroutine 1 [running]:
github.com/daytonaio/daytona/pkg/cmd/workspace.init.func1(0xc000559100?, {0x5499200, 0x0, 0x2dfb03e?})
	/home/runner/work/daytona-release/daytona-release/pkg/cmd/workspace/code.go:102 +0x73a
github.com/spf13/cobra.(*Command).execute(0x53fcae0, {0x5499200, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xab1
github.com/spf13/cobra.(*Command).ExecuteC(0x53f3840)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/daytonaio/daytona/pkg/cmd.Execute()
	/home/runner/work/daytona-release/daytona-release/pkg/cmd/cmd.go:62 +0x44b
main.main()
	/home/runner/work/daytona-release/daytona-release/cmd/daytona/main.go:26 +0x1f


```
**To Reproduce**
Steps to reproduce the behavior:
1. run `daytona code`
2. choose workspace name from UI
4. See error

**Expected behavior**
It should normally open the IDE in the same way as providing the WSname argument `daytona code <WSname>`

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Daytona Version: v0.22.0

--- a/pkg/cmd/workspace/code.go
+++ b/pkg/cmd/workspace/code.go
@@ -50,7 +50,7 @@ var CodeCmd = &cobra.Command{
 		}
 
 		if len(args) == 0 {
-			workspaceList, res, err := apiClient.WorkspaceAPI.ListWorkspaces(ctx).Execute()
+			workspaceList, res, err := apiClient.WorkspaceAPI.ListWorkspaces(ctx).Verbose(true).Execute()
 			if err != nil {
 				log.Fatal(apiclient_util.HandleErrorResponse(res, err))
 			}
@@ -99,13 +99,15 @@ var CodeCmd = &cobra.Command{
 		views.RenderInfoMessage(fmt.Sprintf("Opening the project '%s' from workspace '%s' in %s", projectName, *workspace.Name, ideName))
 
 		providerMetadata := ""
-		for _, project := range workspace.Info.Projects {
-			if *project.Name == projectName {
-				if project.ProviderMetadata == nil {
-					log.Fatal(errors.New("project provider metadata is missing"))
+		if workspace.Info != nil {
+			for _, project := range workspace.Info.Projects {
+				if *project.Name == projectName {
+					if project.ProviderMetadata == nil {
+						log.Fatal(errors.New("project provider metadata is missing"))
+					}
+					providerMetadata = *project.ProviderMetadata
+					break
 				}
-				providerMetadata = *project.ProviderMetadata
-				break
 			}
 		}
 

